pcsx2-qt/Settings/GraphicsSettingsWidget.ui

```diff
@@ -143,7 +143,7 @@
        <item row="3" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
-          <string>De-interlacing:</string>
+          <string>Deinterlacing:</string>
```

Comment:
De-interlacing is actually what i changed it to, people just say deinterlacing for convenience.

Reply:
There is still a mix in other strings between "De-interlacing" and "Deinterlacing". A choice has to be made.

Reply:
Deinterlacing is correct, it's fine https://en.wikipedia.org/wiki/Deinterlacing